bitnami/redis/templates/scripts-configmap.yaml

```diff
@@ -241,20 +241,19 @@ data:
     for node in $( getent ahosts "$HEADLESS_SERVICE" | grep -v "^${myip}" | cut -f 1 -d ' ' | uniq ); do
         info "Cleaning sentinels in sentinel node: $node"
         if is_boolean_yes "$REDIS_SENTINEL_TLS_ENABLED"; then
-            redis-cli {{- if .Values.auth.enabled }} -a $REDIS_PASSWORD {{- end }} -h $node -p {{ .Values.sentinel.service.sentinelPort }} --tls --cert ${REDIS_SENTINEL_TLS_CERT_FILE} --key ${REDIS_SENTINEL_TLS_KEY_FILE} --cacert ${REDIS_SENTINEL_TLS_CA_FILE} sentinel reset "*"
+            redis-cli {{- if .Values.auth.enabled }} -a $REDIS_PASSWORD {{- end }} -h $node -p {{ .Values.sentinel.containerPort }} --tls --cert ${REDIS_SENTINEL_TLS_CERT_FILE} --key ${REDIS_SENTINEL_TLS_KEY_FILE} --cacert ${REDIS_SENTINEL_TLS_CA_FILE} sentinel reset "*"
         else
-            redis-cli {{- if .Values.auth.enabled }} -a $REDIS_PASSWORD {{- end }} -h $node -p {{ .Values.sentinel.service.sentinelPort }} sentinel reset "*"
+            redis-cli {{- if .Values.auth.enabled }} -a $REDIS_PASSWORD {{- end }} -h $node -p {{ .Values.sentinel.containerPort }} sentinel reset "*"
         fi
         sleep {{ .Values.sentinel.cleanDelaySeconds }}
     done
     info "Sentinels clean up done"
 
-
     sentinel_conf_set "sentinel monitor" "{{ .Values.sentinel.masterSet }} "$REDIS_MASTER_HOST" "$REDIS_MASTER_PORT_NUMBER" {{ .Values.sentinel.quorum }}"
 
     add_replica() {
         if [[ "$1" != "$REDIS_MASTER_HOST" ]]; then
-            sentinel_conf_add "sentinel known-replica {{ .Values.sentinel.masterSet }} $1 {{ .Values.sentinel.service.port }}"
```

Comment:
Is this wrong?  Shouldn't this be .Values.sentinel.service.port?

Reply:
Or maybe it should .Values.replica.containerPort

Reply:
Previously, `sentinelPort` was set to `26379` by default, using the same value as `sentinel.containerPort`. In that sense, I don't think this has changed value. Are you finding any particular issues?

Reply:
The port that is set in the "sentinel known-replica" used to be 6379.  After I got this change, the port changed to 26379, which caused a bunch of weird issues (i.e. somehow in a 3 pod setup, the sentinel container in one of the redis pods thought there were 4 sentinels somehow).  Once I changed it back to 6379, everything worked as expected.